support-frontend/assets/pages/digital-subscription-checkout/components/digitalCheckoutForm.jsx

```diff
@@ -182,6 +184,8 @@ class DigitalCheckoutForm extends Component<PropTypes, StateTypes> {
     const offerOnSelected = getAppliedPromoDescription(props.billingPeriod, productPrice);
     const helperSelected = getPriceDescription(productPrice, props.billingPeriod);
     const priceSummary = `${offerOnSelected || 'Enjoy your digital pack free for 14 days, then'} ${helperSelected}.`;
+    const submissionErrorHeading = props.submissionError === 'personal_details_incorrect' ? 'Failed to Create Subscription' :
+      'Payment Attempt Failed';
```

Comment:
I'm not sure if the word "failed" is best fractice.  Perhaps more like "Sorry, we could not create your subscription" and "Sorry, we could not take your payment"

Reply:
I agree actually, will use something more user friendly